docs/posts/signal-number-registration-update.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2022-11-10
+categories:
+    - Software
+authors:
+    - matchboxbananasynergy
+tags:
+    - PSA
+    - Signal
+    - Instant Messengers
+links:
+    - Signal Configuration Guide: signal-configuration-and-hardening.md
```

Comment:
oops my bad, it's supposed to be like https://github.com/privacyguides/blog.privacyguides.org/blob/main/docs/posts/firefox-privacy.md

```suggestion
    - posts/signal-configuration-and-hardening.md
```